radicle-crdt/src/change.rs

```diff
@@ -0,0 +1,130 @@
+use std::collections::BTreeMap;
+
+use radicle_crypto::{PublicKey, Signature, Signer};
+use serde::{Deserialize, Serialize};
+
+use crate::clock::LClock;
+
+/// Identifies a change.
+pub type ChangeId = (LClock, Author);
+/// The author of a change.
+pub type Author = PublicKey;
+/// Alias for `Author`.
+pub type ActorId = PublicKey;
```

Comment:
Why have both `Author` and `ActorId`? :thinking:

Reply:
Lol will remove one.